### PR TITLE
add bool in Writeln

### PR DESCRIPTION
**Pull request by 12mohaned**
Simpler version of previous [PR](https://github.com/chapel-lang/chapel/pull/17381)

### CODE REVIEW

**Comment by 12mohaned** [comment]
@mppf I believe this ready to be merged.

**Comment by bradcray** [comment]
@mppf: Did you ever get a chance to review this?

File: modules/standard/ChapelIO.chpl [hunk 1]
```diff
@@ -834,12 +834,19 @@ module ChapelIO {
   }
 
   /* Equivalent to ``try! stdout.write``. See :proc:`IO.channel.write` */
-  proc write(const args ...?n) {
-    try! stdout.write((...args));
+  proc write(const args ...?n):bool{
+    try! 
+    {
+    return stdout.write((...args));
```
Comment (mppf) [inline modules/standard/ChapelIO.chpl:840]: let's write it with this indenting here and in writeln
``` chapel
try! {
  return stdout.write((...args));
}
```

File: test/library/standard/IO/IOWrite.chpl [hunk 2]
```diff
@@ -0,0 +1,10 @@
+use IO;
+
+proc PrintIOWrite(const input){
+ write(input);
+ writeln(input);
```
Comment (mppf) [inline test/library/standard/IO/IOWrite.chpl:5]: this test needs to check that these functions return `true`. You could use `assert` for that or just print out the return value.

File: test/library/standard/IO/IOWrite.chpl [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+}
+PrintIOWrite(12);
+PrintIOWrite(12.5);
+PrintIOWrite("Hello");
```
Comment (mppf) [inline test/library/standard/IO/IOWrite.chpl:10]: Let's put a newline at the end of the file

**Review by mppf — COMMENTED** [review]
@12mohaned - are you able to follow up? See my comments below.

**Comment by 12mohaned** [comment]
I believe something wrong happened, after pushing the changes I found a new pull request is created, so should I think I should close this one.

**Comment by mppf** [comment]
OK, we'll move this over to the new PR, #17967. Closing this one.

**Pull request closed without merging**